### PR TITLE
"val" in variable names to "valid"

### LINKED ISSUE #275: [Refactor] Variable name consistency-"val" in variable names to "valid"
- Shall we unify the validation abbreviation in docs/variables/functions from `val` to `valid` according to [fastai](https://docs.fast.ai/dev/abbr.html)? Some in PyKale use `val` but others use `valid`.

- Modules that will be modified
image

- Impact on speed - Nope

- Will this be a breaking change? -Nope

### PR DESCRIPTION
**Pull request by XianyuanLiu**
Fixes #275.

### Description
1. Change "val" in variable names to "valid".

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] In-line docstrings updated.
- [ ] [Source for documentation at `docs`](https://github.com/pykale/pykale/tree/main/docs/source) manually updated for new API.

### CODE REVIEW

**Review by sz144 — APPROVED** [review]
The changes in image_access.py, domain_adapter.py, multi_domain_adapter.py and their tests look good to me.

File: examples/bindingdb_deepdta/main.py [hunk 1]
```diff
@@ -37,20 +37,20 @@ def main():
...
     # ---- set model ----
     model = get_model(cfg)
 
     # ---- training and evaluation ----
     gpus = 1 if device == "cuda" else 0
-    checkpoint_callback = ModelCheckpoint(monitor="val_loss", mode="min")
+    checkpoint_callback = ModelCheckpoint(monitor="valid_loss", mode="min")
     trainer = pl.Trainer(max_epochs=cfg.SOLVER.MAX_EPOCHS, gpus=gpus, logger=tb_logger, callbacks=[checkpoint_callback])
-    trainer.fit(model, train_dataloader=train_loader, val_dataloaders=val_loader)
+    trainer.fit(model, train_dataloader=train_loader, valid_dataloaders=valid_loader)
```
Comment (RaivoKoot) [inline examples/bindingdb_deepdta/main.py:53]: pytorch lightning trainer.fit() parameter cannot be called `valid_dataloaders`. Must be called `val_dataloaders`. See [this](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#fit).

**Review by RaivoKoot — COMMENTED** [review]
The one commented line must be fixed. Otherwise good.

File: examples/bindingdb_deepdta/main.py (hunk 1 above)
Comment (XianyuanLiu) [reply examples/bindingdb_deepdta/main.py]: Thanks, Raivo. You are right. I made a mistake. It is fixed now.

**Comment by pz-white** [comment]
The changes in bindingdb_deepdta, drug_gripnet examples, deep_dti pipeline and tests all look good to me. Thanks Xianyuan!

**Review by pz-white — APPROVED** [review]

**Pull request merged**